### ISSUE TITLE
@code does not render tab view as expected

### ISSUE THREAD

**Issue by no0bsec** [getcursor/cursor#685]
Why does using only @code not render the tab view? For example in the following screenshot, only @code is used but no tab is shown.
![image](https://github.com/getcursor/cursor/assets/45028098/b047146d-ad93-4d65-a805-3d525612be21)

However, 'file' and 'docs' work well to show the tabs.
![image](https://github.com/getcursor/cursor/assets/45028098/819ac020-fbf1-4c03-952a-d3329ac7628a)

Did I miss any steps for @code to render the tab view properly?

**Comment by truell20** [comment]
Apologies about this. Want to fix this or find a solution.

To clarify and help me debug, when you type @code and hit enter, nothing shows up?

**Comment by no0bsec** [comment]
Yes, when I type '@code' and hit enter, nothing shows up. Only a tiny part of the tab view UI is visible at the very top, as shown in the screenshot above. This happens with both CTRL+L and CTRL+K shortcuts.
Here is another screenshot showing the CTRL+K behavior:
![image](https://github.com/getcursor/cursor/assets/45028098/c3a4d9f2-1fc4-4d1e-aabd-ca9e0d6af646)

**Comment by no0bsec** [comment]
Hello, thank you for responding to my issue. First I'd like to apologize for the confusion caused by my incomplete environment setup. Through testing and analysis, I realized this problem is related to IntelliSense support.

I did comparison tests on js and python files before and after installing the python extension. I infer that the @code tag relies on VSCode's IntelliSense to display the tab and enable search functions. So when IntelliSense is not enabled for some languages, @code also fails to show the tab.

After properly installing the extensions, I think my question is resolved. Additionally, I have a perhaps unprofessional suggestion - would it be possible to decouple the @code tab from the IntelliSense dependency? Anyway, thank you again for working on this issue.

![image](https://github.com/getcursor/cursor/assets/45028098/5bc8d265-b00e-4e56-888c-ab8d96b7132c)
